po/cs.po

```diff
@@ -2892,9 +2892,9 @@ msgid ""
 "  \\\\\\\\\\\\\\__o\n"
 "__\\\\\\\\\\\\\\'/_"
 msgstr ""
-"   \\\\\\\\\\\n"
-"  \\\\\\\\\\\\\\__o\n"
-"__\\\\\\\\\\\\\\'/_"
+"  (o)(o)--."
+"   \oo/ (  )"
```

Comment:
```suggestion
"   \\oo/ (  )\n"
```
Inside a Cstring a literal BackSlash (\\) needs to be escaped `\\`.
2nd line needs a NewLine char `\n` appended.